.teamcity/_self/projects/WebApp.kt

```diff
@@ -1062,9 +1062,77 @@ fun e2ePreReleaseBuildType( targetDevice: String, buildUuid: String ): E2EBuildT
 		buildUuid = buildUuid,
 		buildName = "Pre-Release E2E Tests ($targetDevice)",
 		buildDescription = "Runs a pre-release suite of E2E tests against trunk on staging, intended to be run after PR merge, but before deployment to production. Will run on $targetDevice size.",
-		testGroup = "calypso-pr",
+		testGroup = "calypso-release",
```

Comment:
I'm using the same test group as in the [PreReleaseE2ETests object](https://github.com/Automattic/wp-calypso/blob/8b023814af41822407a233c8ace13217f39ded15/.teamcity/_self/projects/WebApp.kt#L941).